src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,24 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
+    private static final Comparator<Employee> AGE_NAME_COMPARATOR = new Comparator<>() {
```

Comment:
Don't use anonymous classes - either create a separate class file for comparator or use lambda expression.